backend/graph/service/student_laboratory.go

```diff
@@ -0,0 +1,56 @@
+package service
+
+import (
+	"student-laboratory-matching-app/db"
+	"student-laboratory-matching-app/graph/model"
+
+	"gorm.io/gorm"
+)
+
+type IStudentLaboratoryService interface {
+	ThumbsupToLaboratory(id string) (*model.StudentLaboratory, error)
+}
+
+type studentLaboratoryService struct {
+	db *gorm.DB
+}
+
+func (sls *studentLaboratoryService) ThumbsupToLaboratory(id string) (*model.StudentLaboratory, error) {
```

Comment:
これもLikeでいいんじゃねとは思うけど...

```suggestion
func (sls *studentLaboratoryService) LikeFromStudent(id string) (*model.StudentLaboratory, error) {
```

Reply:
あと中間テーブルのIDはもらえないていで作ってほしい
（初めてのいいねでは中間テーブル生成になるから）